src/app/components/common/walletToggle/index.tsx

```diff
@@ -14,8 +11,8 @@ import { ContainedButton } from "../buttons/containedButton";
 export const WalletToggle = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const isConnecting = useSelector(selectIsConnectingToWallet);
-  const account = useSelector(selectAccount);
+  const isConnecting = useSelector(Web3Selectors.selectIsConnectingToWallet);
+  const account = useSelector(Web3Selectors.selectAccount);
   const walletName: ConnectorPayload["walletName"] = "MetaMask";
```

Comment:
Should we have a set of wallet names defined on a constant? rather than just a simple string?